hyperdash/client.py

```diff
@@ -80,3 +80,6 @@ def iter(self, n, log=True):
                          i, log=False, is_internal=True)
             yield i
             i += 1
+
+    def end(self):
+        self.logger.warning("end() call is unneccessary while using decorator syntax.")
```

Comment:
nit: newline at EOF

Reply:
Actually is this method even necessary? I see what you're trying to do but it might just make things more confusing because your editor might suggest end as a possible method

Reply:
Imagine someone doing a model experiment on hyperdash for 2 weeks.

Under the current paradigm, if they call .end() after those 2 weeks, it will crash their python script. Saving the post processing is worth the warning message imo.

Reply:
ok